.github/workflows/runtests.yml

```diff
@@ -172,7 +172,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.9', '3.10', '3.11', 'pypy-3.8']
+        python-version: ['3.9', '3.10', '3.11', 'pypy3.8-7.3.11']
```

Comment:
In a week or so it won't be necessary to pin the pypy version to 7.3.11 like this but since it is a new release it hasn't rolled out by default to setup-python.